01-intro.Rmd

```diff
@@ -16,15 +16,37 @@ First we will discuss the motivations behind the new data management and sharing
 
 So why is the NIH doing this? There are several reasons why sharing data can be beneficial to the scientific community.
 
-- **Supports transparency** - Sharing data provides more clarity about how studies are performed. 
--  **Encourages reproducibility and rigor** - Having the data accessible, allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
+- **Supports transparency** - Sharing data provides more clarity about how studies are performed. Many scientists also believe in an ethical responsibility to study participants (@Bauchner_citation_2016).
```

Comment:
Nice! Yeah, I think we could add more references...

Reply:
What do you think about a page of citations that they can refer to in the plan?

Reply:
oooh cool idea